### ISSUE TITLE
java.lang.NoClassDefFoundError: nl/basjes/shaded/com/google/common/net/InternetDomainName

### ISSUE THREAD

**Issue by andreitsibets** [nielsbasjes/yauaa#107]
Hi, 
I've faced the following issue:
`java.lang.NoClassDefFoundError: nl/basjes/shaded/com/google/common/net/InternetDomainName`

I see in the pom file: 
```
<includes>
<include>org.antlr:antlr4-runtime</include>
<include>org.springframework:spring-core</include>
<include>org.yaml:snakeyaml</include>
</includes>
```
but relocation is for 4 artifacts:
```
<relocations>
<relocation>
<pattern>org.springframework</pattern>
<shadedPattern>nl.basjes.shaded.org.springframework</shadedPattern>
</relocation>
<relocation>
<pattern>org.antlr</pattern>
<shadedPattern>nl.basjes.shaded.org.antlr</shadedPattern>
</relocation>
<relocation>
<pattern>org.yaml.snakeyaml</pattern>
<shadedPattern>nl.basjes.shaded.org.yaml.snakeyaml</shadedPattern>
</relocation>
<relocation>
<pattern>com.google</pattern>
<shadedPattern>nl.basjes.shaded.com.google</shadedPattern>
</relocation>
</relocations>
```

I guess it causes the issue.

**Comment by nielsbasjes** [comment]
Yes you are right.
Today someone else also reported this problem and I have already fixed it.
Ill get a release out very soon.

**Comment by nielsbasjes** [comment]
I released version 5.2 that fixes this serious issue.
